src/test/java/com/google/api/codegen/testdata/csharp/csharp_library.baseline

```diff
@@ -2842,6 +2842,60 @@ namespace Google.Example.Library.V1.Tests
             mockGrpcClient.VerifyAll();
         }
 
+        [Fact]
+        public void CreateShelf2()
+        {
+            Mock<LibraryService.LibraryServiceClient> mockGrpcClient = new Mock<LibraryService.LibraryServiceClient>(MockBehavior.Strict);
+            mockGrpcClient.Setup(x => x.CreateLabelerClient())
+                .Returns(new Mock<Labeler.LabelerClient>().Object);
+            mockGrpcClient.Setup(x => x.CreateOperationsClient())
+                .Returns(new Mock<Operations.OperationsClient>().Object);
+            Shelf expectedResponse = new Shelf
+            {
+                ShelfName = new ShelfName("[SHELF_ID]"),
+                Theme = "theme110327241",
+                InternalTheme = "internalTheme792518087",
+            };
+            // TODO: Add verification of request object
```

Comment:
Any reason not to do this right now? We expect the *exact* request object, right?

Reply:
I'm now working on a PR to verify the request object in every case including this trivial case; is it OK to not make this change here?